src/olympia/scanners/admin.py

```diff
@@ -436,6 +440,15 @@ def formatted_matched_rules_with_files(
 
     formatted_matched_rules_with_files.short_description = 'Matched rules'
 
+    def formatted_score(self, obj):
+        if obj.scanner not in [CUSTOMS, MAD]:
+            return '-'
+        if obj.score < 0:
+            return 'n/a'
```

Comment:
Why differentiate between `-` and `n/a`? Wouldn't `n/a` be appropriate for the case where `obj.scanner not in [CUSTOMS, MAD]`?

Reply:
So that's because some scanners won't have a score at all whereas CUSTOMS/MAD should (at least the ML API should return something, which could be `-1`)

Reply:
I gathered that from the code, but that doesn't explain why we need both `-` and `n/a`. Will a user care about the difference between `-` and `n/a`? Is there a meaningful difference between them from a user perspective?

Reply:
Maybe not. To me the distinction makes sense because I can see which results are analyzed by the ML service and if I see `n/a` next to a `customs` result, I'll have to investigate. It would likely work for "-" too, I guess.

Reply:
I keep your comment in mind as I am sure we'll have to update how we display the score once the integration is enabled in production.